packages/gatsby/src/utils/handle-flags.ts

```diff
@@ -26,7 +26,11 @@ const handleFlags = (
   // Prepare config flags.
   // Filter out any flags that are set to false.
   const availableFlags = new Map()
-  flags.forEach(flag => availableFlags.set(flag.name, flag))
+  flags.forEach(flag => {
+    if (flag.testFitness(flag)) {
+      availableFlags.set(flag.name, flag)
+    }
```

Comment:
This can result in 
```
warning The following flag(s) found in your gatsby-config.js are not known:
- LAZY_IMAGES
```
due to unknown flags checking for `availableFlags` just below
![Screenshot 2020-12-10 at 19 18 34](https://user-images.githubusercontent.com/419821/101813088-8fa4d700-3b1c-11eb-8046-6c8f7a5580d1.png)

Reply:
Flag is not unknown - just not "fit" :)

Reply:
fixed in https://github.com/gatsbyjs/gatsby/pull/28387/commits/51b4eaf614f43c4ab3d452cdff2d881be555e15e